website/docs/r/schema_template_anp_epg_subnet.html.markdown

```diff
@@ -13,16 +13,17 @@ Manages MSO Schema Template Application Network Profiles Endpoint Groups Subnets
 ## Example Usage ##
 
 ```hcl
+
 resource "mso_schema_template_anp_epg_subnet" "subnet1" {
-  schema_id = "5c6c16d7270000c710f8094d"
-  anp_name = "WoS-Cloud-Only-2"
-  epg_name ="EPG4"
-  template = "Template1"
-  ip = "31.101.102.0/8"
-  scope = "public"
-  shared = true
-  
+  schema_id = mso_schema.schema1.id
+  anp_name  = "WoS-Cloud-Only-2"
+  epg_name  = "EPG4"
```

Comment:
should refer to mso_schema_template_anp_epg

Reply:
ok